espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Server.java

```diff
@@ -34,11 +35,11 @@ public Server() throws IOException, DuplicateRouteException {
     }
 
     @Override
-    public Response serve(IHTTPSession session) {
+    public Response serve(String uri, Method method, Map<String, String> headers, Map<String, String> parms, Map<String, String> files) {
```

Comment:
I assume in strictly-typed Java world only argument types matter, but you can use your own names for override method arguments. 
https://stackoverflow.com/questions/9627139/overriding-methods-with-different-variable-names